packages/design-system/src/styles/components/_Autocomplete.scss

```diff
@@ -5,7 +5,7 @@
 // Autocomplete styles extend Dropdown styles
 
 .ds-c-autocomplete {
-  @extend .ds-c-dropdown;
+  @extend %dropdown;
```

Comment:
For some reason this just doesn't work, and I haven't been able to figure out why. Sass never ends up creating a `.ds-c-autocomplete` rule in Storybook, though it seems to exist in the CSS file generated by the core build. Like you saw locally, it doesn't matter whether Storybook is built statically or served dynamically; the Sass just doesn't match. Until we can figure this out, my recommendation is to revert the changes and allow `@extend` in the config.